ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue

```diff
@@ -600,13 +600,24 @@ export default defineComponent({
 
 <style lang="scss" scoped>
 @import '@/assets/styles/theme.scss';
+::v-deep .theme--light.v-icon.mdi-close {
+  color: $primary-blue !important;
+}
+
+.solid {
+  border: 0;
+  border-top: 0.25px solid $gray4 !important;
+}
```

Comment:
The border top works for sure, just know you have options with `v-divider` to do a similar thing with more flexibility.